nntrainer/models/neuralnet.cpp

```diff
@@ -447,8 +448,13 @@ void NeuralNetwork::setBatchSize(unsigned int batch) {
   model_graph.setBatchSize(batch);
   manager->setBatchSize(batch);
 
-  if (data_buffer && data_buffer->setBatchSize(batch_size) != ML_ERROR_NONE)
-    throw std::invalid_argument("Error setting batchsize for the dataset");
+  for (auto &db : data_buffers) {
+    if (db != nullptr) {
+      int status = db->setBatchSize(batch_size);
+      ml_loge("[model] setting batchsize from data buffer failed");
```

Comment:
missing `status != ML_ERROR_NONE`?

Reply:
it's throwing status at line 455 which is being captured at exception boundary

Reply:
ml_loge is logging always even though the there was no error.

Reply:
Right, I'll change

Reply:
Done